### PR TITLE
Sanity check

### PR DESCRIPTION
**Pull request by smolix**
*Description of changes:*

Attention - mostly checking whether it builds. 

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by astonzhang** [comment]
Thanks Alex! I'm merging it and will add edits shortly

File: chapter_attention-mechanisms-and-transformers/attention-pooling.md [hunk 1]
```diff
@@ -54,349 +38,152 @@ from d2l import torch as d2l
...
-we can see that the closer the query-key pair is,
-the higher attention weight is in the attention pooling.
+Let's have a look at the kind of estimates that the different kernels produce.
 
 ```{.python .input}
-%%tab all
-d2l.show_heatmaps([[attention_weights]],
-                  xlabel='Sorted training inputs',
-                  ylabel='Sorted validation inputs')
+def plot_results(x_train, y_train, x_test, y_test, kernels, names, attention = False):
```
Comment (AnirudhDagar) [inline chapter_attention-mechanisms-and-transformers/attention-pooling.md:126]: I think we need to add back other frameworks as well. MxNet and Tensorflow are removed and are broken in the current state.

**Review by AnirudhDagar — COMMENTED** [review]
@astonzhang @smolix Not using a tab mark results in only running the pytorch cell because that is primary tab. Other frameworks are ignored now. I'll try to send a quick fix now, since this is blocking jax release.

**Comment by AnirudhDagar** [comment]
See http://preview.d2l.ai/d2l-en/master/chapter_attention-mechanisms-and-transformers/attention-pooling.html#attention-pooling-via-nadaraya-watson-regression for results only showing pytorch.

**Comment by AnirudhDagar** [comment]
I've made the fix [here](https://github.com/d2l-ai/d2l-en/pull/2379/commits/5f8434406b66ebd3a229bd06e30a25439fcc0ade). We will include that once JAX branch is merged.

**Pull request merged**